src/status_im/contexts/settings/wallet/keypairs_and_accounts/view.cljs

```diff
@@ -93,7 +93,11 @@
         profile-picture               (rf/sub [:profile/image])
         customization-color           (rf/sub [:profile/customization-color])
         {missing-keypairs  :missing
-         operable-keypairs :operable} (rf/sub [:wallet/settings-keypairs-accounts])]
+         operable-keypairs :operable} (rf/sub [:wallet/settings-keypairs-accounts])
+        on-import-click               (rn/use-callback #(rf/dispatch [:open-modal
+                                                                      :screen/settings.scan-key-pair-qr
+                                                                      missing-keypairs])
+                                                       [missing-keypairs])]
```

Comment:
When we implement the single key-pair import logic, can we re-use this code and only dispatch one missing-keypair? If so that would be great 🙌

Reply:
Yup defiantly will use re-use these codes as the func already takes array of key-uid. so can be one or multiple.